app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -54,6 +55,12 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
     private val token = Token(LeanplumId, LeanplumToken)
 
     override fun start() {
+        val currentLocale = Locale.getDefault().isO3Language
```

Comment:
I have definitely used `Locale.getDefault`, many times in our codebase, but I _believe_ we should be using `LocaleManager.getCurrentLocale` here.  I know that the `LocaleManager` will be updated when a user changes language in-app, and I'm not sure whether or not `Locale.getDefault` will.

Reply:
Testing `Locale.getDefault().isO3Language` I saw it returning the expected language (system default/app set language) but I agree that being more explicit and also having more certainty about the result is better.
Implemented this and rebased.
Thanks!